mysql-test/suite/sys_vars/t/clone_dscp_on_socket_basic.test

```diff
@@ -0,0 +1,125 @@
+##################### dscp_on_socket_basic.test ###############################
```

Comment:
Plugin sys var tests do no go to the `sys_var` suite, usually there are just functional tests added as needed in the specific suites (`clone` and/or `rocksdb_clone` here). In this case a functional test is not possible if I understand correctly, so no need for this test. You can check by running MTR `sys_vars.all_vars` test and if it fails, checking its failure output for any mentions of this sysvar.

In the case this test is needed, it looks like one of the verbose templates for a sysvar test has been used here. It is OK to leave it as-is, but in general these tests can be brief and with fewer comments